src/html/settings.html

```diff
@@ -4,346 +4,399 @@
+            <circle cx="11.5" cy="11.5" r="8.5" fill="#fff"/>
+            <path d="M11.1 17c-3 0-5.4-2.5-5.4-5.5 0-2.5 1.7-4.6 4-5.2v1.6c-1.5.5-2.5 2-2.5 3.6 0 2.2 1.7 4 3.9 4s4-1.8 4-4c0-1.6-1.1-3.1-2.5-3.6V6.3c2.3.6 4 2.7 4 5.2 0 3-2.5 5.5-5.5 5.5zm-.8-12.5h1.6v7.7h-1.6V4.5zm.8-4.4C5 .1 0 5.1 0 11.3c0 6.1 5 11.1 11.1 11.1 6.2 0 11.1-5 11.1-11.1C22.2 5.1 17.3.1 11.1.1z" fill="#e20505"/>
+          </g>
+        </svg>
       </div>
-      <div class="tabs">
-        <div class="tab" data-tab="general">
-          <div class="fields">
-            <div class="field">
-              <div class="field__label">
-                <label for="default-project">Default Project</label>
-                <div class="description">The default project will be selected when timer is started with no matched project</div>
-              </div>
-
-              <div>
-                <div id="default-project-container">
-                </div>
-              </div>
-            </div>
-
-            <div class="field">
-              <div class="field__label">
-                <label for="remember-project-per">Remember last chosen project</label>
-                <div class="description">Remembers the last project you used on each integration or URL. Overrides the default project.</div>
-              </div>
-
-              <div>
-                <select id="remember-project-per">
-                  <option value="false">Use global default</option>
-                  <option value="service">Service</option>
-                  <option value="url">URL</option>
-                </select>
-              </div>
-            </div>
-
-            <div class="field">
-              <div class="field__label">
-                <label for="enable-auto-tagging">Enable automatic tagging</label>
-                <div class="description">If an integration supports tags or labels, automatically add them to the timer and
-                  create them in Toggl. Not available in all integrations.</div>
-              </div>
-
-              <div>
-                <input type="checkbox" id="enable-auto-tagging">
-              </div>
+      <div class="section">
+        <h3>Button Settings</h3>
```

Comment:
`Button` seems redundant here, maybe just use `Settings`

Reply:
If we remove it I'd like some other way to show clearly that this is a button page, not a webapp page. Should we talk to product about this?